suite/spdz/pom.xml

```diff
@@ -15,11 +15,18 @@
   <name>fresco-spdz</name>
 
   <dependencies>
+  
     <dependency>
       <groupId>dk.alexandra.fresco</groupId>
       <artifactId>core</artifactId>
       <version>${fresco-version}</version>
     </dependency>
+    
+    <dependency>
+      <groupId>dk.alexandra.fresco</groupId>
+      <artifactId>mascot</artifactId>
```

Comment:
I do not agree that the online spdz should depend on mascot. Cause it doesn't :) - mascot effectively just provides a different SpdzDataSupplier, and all tests on that should be done within mascot. The same was done with the (now removed) fuelstation where a REST data supplier was created and used only within that project. So I suggest moving the mascot supplier to mascot.

Reply:
There are three options:
1. Mascot depending on spdz
1. Spdz depending on mascot 
1. new spdz offline maven project

1 is your suggestion, and implies that mascot should be seen as a subprotocol of spdz
2 is the current solution
3 is the correct, but cumbersome solution

The plan is to keep mascot pure, therefore we went with 2. Which also eliminates the potential jar hell at the end of that choice.